vscode/src/completions/inline-completion-item-provider.ts

```diff
@@ -254,9 +258,18 @@ export class InlineCompletionItemProvider implements vscode.InlineCompletionItem
             // latency so that we don't show a result before the user has paused typing for a brief
             // moment.
             if (result.source !== InlineCompletionsResultSource.LastCandidate) {
-                const minimumLatencyFlag = await minimumLatencyFlagsPromise
-                if (triggerKind === TriggerKind.Automatic && minimumLatencyFlag) {
+                const [userLatencyFlag, languageLatencyFlag, providerLatencyFlag] =
+                    await Promise.all(minLatencyFlagsPromises)
+                const latencyFeatureFlags: LatencyFeatureFlags = {
+                    user: userLatencyFlag,
+                    language: languageLatencyFlag,
+                    provider: providerLatencyFlag,
+                }
+
+                const isMinLatencyEnabled = userLatencyFlag || languageLatencyFlag || providerLatencyFlag
```

Comment:
...continued from above.
```suggestion
                const latencyFeatureFlags: LatencyFeatureFlags = {
                    user: await minLatencyFlagsPromises.user,
                    language: await minLatencyFlagsPromises.language,
                    provider: await minLatencyFlagsPromises.provider,
                }

                const isMinLatencyEnabled = latencyFeatureFlags.user || latencyFeatureFlags.language || latencyFeatureFlags.provider
```